src/main/resources/assets/armorplus/lang/zh_cn.lang

```diff
@@ -385,14 +397,57 @@ item.armorplus.redstone_flux_shovel.name=红石通量锹
 
 #Entities
 entity.ender_dragon_zombie.name=末影龙僵尸
-entity.ice_golem.name=雪傀儡 (WIP)
+entity.ice_golem.name=雪傀儡（未完成，实验阶段）
+entity.overlord_of_the_guardians.name=管理者（未完成，实验阶段）
+entity.skeletal_king.name=骷髅王（未完成，实验阶段）
+entity.freeze_bomb.name=霜冻介质（未完成，远程，实验阶段）
+entity.wither_minion.name=凋谢仆从（未完成，远程，Spawner）
```

Comment:
凋零仆从（未完成，远程，刷怪笼）？

Reply:
About this word
Originally, I did not translate because if I can not express what it means by the meaning of my translation, I suppose it should have been generated by wither?